ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.11.yaml

```diff
@@ -57,6 +57,12 @@ tests:
   steps:
     cluster_profile: vsphere
     workflow: cucushift-installer-rehearse-vsphere-ipi-baselinecaps-none-additionalenabledcaps
+- as: installer-rehearse-nutanix
+  cluster: build01
```

Comment:
@sgaoshang if we have not set the cluster, then what would happen?

Reply:
I'll have a try without it.

Reply:
@jianlinliu got error without cluster: build01
`INFO[2022-09-05T08:04:14Z] Releasing leases for test installer-rehearse-nutanix 
INFO[2022-09-05T08:04:14Z] Ran for 1m55s                                
ERRO[2022-09-05T08:04:14Z] Some steps failed:                           
ERRO[2022-09-05T08:04:14Z] 
  * could not run steps: step installer-rehearse-nutanix failed: failed to create RBAC objects: could not create role binding "installer-rehearse-nutanix-vpn": clusterroles.rbac.authorization.k8s.io "ci-operator-vpn" not found 
INFO[[20](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/31581/rehearse-31581-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.11-installer-rehearse-nutanix/1566698191821213696#1:build-log.txt%3A20)22-09-05T08:04:14Z] Reporting job state 'failed' with reason 'executing_graph:step_failed:utilizing_lease:executing_test:executing_multi_stage_test:binding_roles'`
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/31581/rehearse-31581-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.11-installer-rehearse-nutanix/1566698191821213696

Reply:
okay. got it. thx.